velox/functions/prestosql/tests/UuidFunctionsTest.cpp

```diff
@@ -108,5 +108,49 @@ TEST_F(UuidFunctionsTest, unsupportedCast) {
       evaluate("cast(123 as uuid())", input), "Cannot cast BIGINT to UUID.");
 }
 
+TEST_F(UuidFunctionsTest, comparisons) {
+  auto data0 = makeFlatVector<std::string>({
+      "33355449-2c7d-43d7-967a-f53cd23215ad",
+      "eed9f812-4b0c-472f-8a10-4ae7bff79a47",
+      "f768f36d-4f09-4da7-a298-3564d8f3c986",
+      "eed9f812-4b0c-472f-8a10-4ae7bff79a47",
+      "f768f36d-4f09-4da7-a298-3564d8f3c986",
+      "33355449-2c7d-43d7-967a-f53cd23215ad",
+  });
+
+  auto data1 = makeFlatVector<std::string>({
+      "00000000-0000-0000-0000-000000000000",
+      "ffffffff-ffff-ffff-ffff-ffffffffffff",
+      "f768f36d-4f09-4da7-0000-000000000000",
+      "eed9f812-4b0c-472f-ffff-ffffffffffff",
+      "a768f36d-4f09-4da7-a298-3564d8f3c986",
+      "33355449-2c7d-43d7-967a-f53cd23215ad",
+  });
+
+  auto actual = evaluate("cast(c0 as uuid) < cast(c1 as uuid)", makeRowVector({data0, data1}));
```

Comment:
Any particular reason not to test one value at a time using evaluateOnce? This makes tests easier to read and debug.

Reply:
No, just not familiar with the test environment. I will try with `evaluateOnce`.